CHANGELOG_UNRELEASED.md

```diff
@@ -101,7 +102,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Replaced the legacy generalised induction idiom with a more robust one
 that does not rely on the `{-2}` numerical occurrence selector, using
 new `ssrnat` helper lemmas `ltn_ind`, `ubnP`, `ubnPleq`,  ...., (see above). The new idiom is documented in `ssrnat`.
-   This change anticipates an expected evolution of `fintype` to integrate `finmap`. It is likely that the new definition of the `#|A|` notation will hide multiple occurrences of `A`, which will break the `{-2}` induction idiom. Client libraries should update before the 1.11 release.
+   This change anticipates an expected evolution of `fintype` to integrate `finmap`. It is likely that the new definition of the `#|A|` notation will hide multiple occurrences of `A`, which will break the `{-2}` induction idiom. Client libraries should update before the 1.11 release (see PR #434 for examples).
+   
+ - Replaced the use of the accidental convertibility between `enum A` and 
+   `filter A (index_enum T)` with more explicit lemmas `big_enumP`, big_enum`, `big_enum_cond`, `big_image` added to the `bigop` library. This convertibilty, and indeed equality may no longer hold in future `mathcomp` releases when sets over `finType`s are generalised to finite sets over `choiceType`s, so client libraraies should stop relying on this convertibility, as well as the `filter_index_enum` lemma. File `bigop.v` has some boilerplate to help with the port; also see PR #441 for examples.
```

Comment:
missing opening `` ` `` before `big_enum`

Reply:
Right; I was expecting markdown to turn the reference into a hyperlink as it does in the comments (viz. #434), but apparently that doesn't work.

Reply:
> Right; I was expecting markdown to turn the reference into a hyperlink as it does in the comments (viz. #434), but apparently that doesn't work.

it works only on github :wink: but the CHANGELOG is destined to be read independently

Reply:
Since we've switched to markdown the CHANGELOG is going to be hard to read outside GitHub anyway.., but it actually doesn't even work when viewing the LOG file on GitHub.